core/src/test/java/nl/nn/adapterframework/util/RunStateTest.java

```diff
@@ -1,31 +0,0 @@
-package nl.nn.adapterframework.util;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-
-import org.junit.Test;
-
-public class RunStateTest {
-
-	@Test
-	public void startedEqualsSTARTED() throws Exception {
-		assertTrue(RunStateEnum.STARTED.isState("Started"));
-	}
-
-	@Test
-	public void compareTo() throws Exception {
-		assertEquals(0, RunStateEnum.ERROR.compareTo(RunStateEnum.ERROR));
-		assertTrue(RunStateEnum.ERROR.compareTo(RunStateEnum.STARTED) < 0);
-		assertTrue(RunStateEnum.ERROR.compareTo(RunStateEnum.STOPPED) < 0);
-		assertTrue(RunStateEnum.STOPPED.compareTo(RunStateEnum.STARTED) > 0);
-	}
-
-	@Test
-	public void checkGetName() throws Exception {
-		assertEquals("Started", RunStateEnum.STARTED.getName());
-		assertEquals("Starting", RunStateEnum.STARTING.getName());
-		assertEquals("Stopped", RunStateEnum.STOPPED.getName());
-		assertEquals("Stopping", RunStateEnum.STOPPING.getName());
-		assertEquals("**ERROR**", RunStateEnum.ERROR.getName());
```

Comment:
Dit zijn / waren de console/api namen. Ik weet niet zeker hoe dat in de frontend geimplementeerd is, misschien dat nu het filteren niet meer werkt oid? Dat zou je nog even kunnen/moeten testen.

Reply:
ik zie in [ConfigurationStatus](https://github.com/ibissource/iaf/blob/master/webapp/src/main/webapp/iaf/gui/views/ShowConfigurationStatus.html#L51) de namen uitgeschreven zijn